book/src/SUMMARY.md

```diff
@@ -1,11 +1,30 @@
 # Table of contents
 
-* [The Solana Codebase](introduction.md)
-* [Building from Source](getting-started/README.md)
-  * [Example Client: Web Wallet](getting-started/webwallet.md)
-* [Programming Model](programs/README.md)
-  * [Example: Tic-Tac-Toe](programs/tictactoe.md)
-  * [Drones](programs/drones.md)
+* [Introduction](introduction.md)
+* [Using Solana](using-solana.md)
```

Comment:
Not a fan of this name (too general) and the content in this section is pretty specialized (offline signing and paper wallet). How about naming it something more specific to the CLI and moving it between "Running an Archiver" and "A Solana Cluster"? Maybe "Using Solana's CLI"?

Reply:
So I'm attempting to funnel people from the most basic usage down to gaining enough expertise to make contributions to the source base.  If we move that section down, the reader jumps straight to application development, which is a smaller audience than those wanting to just transfer tokens.

Good point though, that everything in that section is command-line. I'll rename to something more specific.

Reply:
Fixed in 8ea99a9